test/ts-scenario/steps/lib/channel.ts

```diff
@@ -113,8 +114,10 @@ export async function cli_get_channels(orgName: string, tls: boolean): Promise<s
 		];
 
 		command = command.concat(tlsOptions);
-		const channelNames = await commandRunner.runShellCommand(true, command.join(' '), VERBOSE_CLI) as any;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
I think a better type definition for `CommandRunner.runShellCommand()` should make this go away